studentInfoOGB.py

```diff
@@ -0,0 +1,96 @@
+import random
+
+firstName = ["jay", "jim", "roy", "axel", "billy", "charlie", "jax", "gina", "paul",
+"ringo", "ally", "nicky", "cam", "ari", "trudie", "cal", "carl", "lady", "lauren",
+"ichabod", "arthur", "ashley", "drake", "kim", "julio", "lorraine", "floyd", "janet",
+"lydia", "charles", "pedro", "bradley"]
+
+lastName = ["blake", "holt", "meilijson", "brannon", "fieser", "line", "hellman",
+"lyndon", "wendell", "corr", "fricano", "carrigan", "giddings", "piesse", "charlwood",
+"thaler", "ambrose", "hamer", "schweig", "geiger", "kinder", "rogantin", "chevallier"
+"wang", "tiedemann", "cohn", "digel", "reheem", "dzieciololowski", "callanan",
+"lie", "oden"]
+
+def main():
+  students = [
+    Student("Larsson", 37),
+    Student("BonJovi", 55),
+  ]
+
+  printHeader()
+  selection = int(getUserSelection())
+  if selection == 0:
+    printStudentsByAge(students)
+  elif selection == 1:
+    printStudentsByLName(students)
+  elif selection == 2:
+    printStudentsByFName(students)
+  else:
+    print ("SELECTION NOT RECOGNIZED")
+
+
+class Student:
+  def __init__(self, lastName, age):
```

Comment:
Despite being required parameters, it seems that `lastName` and `age` are never used.

Maybe you intended to make them optional, as in generate a random `lastName` if `None` is supplied? If so then you could use an adaptation of this:

```
class Foo:
  def __init__(self, color=None):
    self.color = self.generateRandomColor() if color is None else color
```